### ISSUE TITLE
gPodder

### ISSUE THREAD

**Issue by anewuser** [portapps/portapps#269]
* Name : gPodder
* Description : podcast client
* Website : https://github.com/gpodder/gpodder/
* License : GPLv3

gPodder already has a portable build, but:
* It is a simple self-extracting 7-Zip executable with a root folder, which is problematic (see [gpodder#1346](https://github.com/gpodder/gpodder/issues/1346))
* To be truly portable, the download folder should be within the application folder. Currently, gPodder has a preconfigured download path and doesn't even offer an easy option to change it. Windows users need to change an environment variable: [gpodder#729](https://github.com/gpodder/gpodder/issues/729)

It'd be nice if a volunteer could offer assistance by providing proper portable options.

**Comment by elelay** [comment]
Yes, please, assistance on the windows side will be welcome :+1:
